.travis.yml

```diff
@@ -1,14 +1,13 @@
 # This Travis job script has been generated by a script via
 #
-#   haskell-ci '--config=cabal.haskell-ci' 'cabal.project'
+#   haskell-ci 'reflex.cabal'
```

Comment:
The problem is that you don't use config file, that's why things go wrong in CI.

Reply:
Thanks!